Sources/MapboxNavigation/Match.swift

```diff
@@ -14,19 +14,18 @@ extension Match {
      - returns: A polyline whose length is twice `distance` and whose centroid is located at the maneuver.
      */
     func polylineAroundManeuver(legIndex: Int, stepIndex: Int, distance: CLLocationDistance) -> LineString {
-        let precedingLegs = legs.prefix(upTo: legIndex)
-        let precedingLegCoordinates = precedingLegs.flatMap { $0.steps }.flatMap { $0.shape?.coordinates ?? [] }
-        
-        let precedingSteps = legs[legIndex].steps.prefix(upTo: stepIndex)
-        let precedingStepCoordinates = precedingSteps.compactMap { $0.shape?.coordinates }.reduce([], +)
-        let precedingPolyline = LineString((precedingLegCoordinates + precedingStepCoordinates).reversed())
+        var precedingCoordinates = [LocationCoordinate2D]()
```

Comment:
This function is used in RouteTests.swift for maneuver arrow, it should share the same logic as the `Route.polylineAroundManeuver(legIndex:stepIndex:distance:)`